charts/jenkins/templates/jenkins-controller-ingress.yaml

```diff
@@ -60,6 +60,7 @@ spec:
             name: {{ template "jenkins.fullname" . }}
             port:
               number: {{ .Values.controller.servicePort }}
+        pathType: ImplementationSpecific
```

Comment:
This is only compatible with k8s 1.18 or later.
kindly, make sure this is back-ward compatible with the kubernetes apis & resources versions supported currently by this chart.
Otherwise, you can add condition :
- `Capabilities.APIVersions.Has $version`
- or `Capabilities.KubeVersion`
- or `Capabilities.KubeVersion.Major` ( compare it with `18`)
- .. so on
ref: https://helm.sh/docs/chart_template_guide/builtin_objects/

Reply:
it's guarded by an if block >= 1.19, 5 lines up